app/frontend/actions.ts

```diff
@@ -962,12 +963,12 @@ export default ({setState, getState}: {setState: SetStateFn; getState: GetStateF
     })
   }
 
-  const calculateDelegationFee = async () => {
+  const calculateDelegationFee = async (): Promise<void> => {
     const state = getState()
     setPoolInfo(state)
-    const poolHash = state.shelleyDelegation.selectedPool.poolHash
+    const poolHash = state.shelleyDelegation.selectedPool.poolHash as string
```

Comment:
I didnt want to change the type of shelley delegation since David is changing it in another PR, and also, this is rather a preparation for the refactor of tx planning. I have WIP PR in which a have the tx plan arguments typed and that's why I had to change it this way. As mentioned, I was doing only nessasarry stuff so I can move on to tx plan refactor with typed arguments.